napari/components/_viewer/viewer.py

```diff
@@ -39,15 +33,19 @@ def __init__(self):
         self._interactive = True
         self._top = None
 
-        self._qt = QtViewer(self)
+        # TODO: this should be eventually removed!
+        # initialised by QtViewer when it is contrsucted by the model
+        self._qtviewer = None
```

Comment:
May i get clarification for this part? a bit confusing.

Reply:
In the future the viewer `model` file will not need to have the viewer `view` file attached to it, but we still need it for now. This change will come in the future refactoring